.github/pull_request_template.md

```diff
@@ -1,12 +1,13 @@
-<!-- Please provide a descriptive title for your Pull Request above.  -->
-
 ## 🚀 Description
 
-<!-- Please provide a description of the changes in your Pull Request, in particular the motivation for the changes. -->
+<!--
```

Comment:
Bonus. Tried to simplify the language slightly throughout to increase the change that any of it will be read. Used lists for the same similar.

Reply:
<img width="654" alt="image" src="https://github.com/user-attachments/assets/77e1e3a3-88c6-443d-af65-52f9aa1a155a" />

Reply:
I dig these changes! 💅